src/utils/useWeb3Connect.js

```diff
@@ -24,14 +24,17 @@ import {
 } from './3BoxManager';
 
 const daiAbi = require('../abis/ERC20.json');
-const daoAbi = require('../abis/NoLossDao.json');
+const daoAbi = require('../abis/NoLossDao_v0.json');
+const depositAbi = require('../abis/PoolDeposits.json');
 
 const SUPPORTED_CHAIN_ID = Number(process.env.REACT_APP_SUPPORTED_CHAIN_ID);
 const SUPPORTED_NETWORK = 'kovan';
 
+const CHAIN_ID = process.env.REACT_APP_DEFAULT_CHAIN_ID || '42';
```

Comment:
I'm not happy with hard-coding this, we should always get the chain id dynamically in the future. This is just the minimal change to get this to work.